### DISCUSSION TITLE
FzfLua for :messages

### DISCUSSION THREAD

**Discussion by zokrezyl** [ibhagwan/fzf-lua#1382] [General]
Hi!

Have been using fzf for decades and migrated from fzf-vim to fzf-lua.

It would be of great use to have fzf for :messages.

Regards
ZZ

**Comment by ibhagwan** [comment]
I wonder if there's an API better than parsing `:messages` that can help retrieve all of the messages inlcuding their priority (info, warn, error, etc).
